windows/src/engine/keyman32/globals.h

```diff
@@ -208,6 +209,7 @@ typedef struct tagKEYMAN64THREADDATA
 
   PKEYMANPROCESSOUTPUTFUNC TIPProcessOutput;
   PKEYMANGETCONTEXTFUNC TIPGetContext;
+  PKEYMANGETCONTEXTISSELECTEDFUNC TIPGetContextEx;
```

Comment:
These names should really match -- should be either:

```suggestion
  PKEYMANGETCONTEXTISSELECTEDFUNC TIPGetContextIsSelected;
```

or

```suggestion
  PKEYMANGETCONTEXTEXFUNC TIPGetContextEx;
```